pyabc/sge/sge.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, tmp_directory: str = None, memory: str = '3G',
             print("Warning: Could not find SGE installation.", file=sys.stderr)
 
         # python interpreter which executes the jobs
-        if type(time_h) is not int:
+        if not isinstance(time_h, int):
```

Comment:
thanks, I will improve the docs here. For some things I'm actually not sure what they do (or would need to look into details), because sge is kind of detached from the rest of pyabc a bit, and I never used it.

Reply:
so will this be addressed now or on hold?

Reply:
I added some things, but left some things untouched. I will look into this in more detail when I need it.